README.md

```diff
@@ -241,7 +241,7 @@ relevant field `exception` and the stack trace therewithin.
 An it-server project has been included as a reference, providing an example page.
 You can see this in action by running the following command:
 ```sbt
-PLAY_VERSION=2.7 sbt "project itServer" run
+PLAY_VERSION=3.0 sbt "project it-server-play-30" run
```

Comment:
@JoPintoPaul just thought, one thing it might be worth adding - a note to say prominently something like  "The latest major version releases have not yet been performance tested, please make sure when updating that you check for regressions before going to prod."

Reply:
@oscarduignan That's excellent advice, do you think that would perhaps be better in the team channels? Wonder if the README will quickly become out-of-date otherwise.